src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -206,6 +211,19 @@ private enum ResponseStatus
             return authParams;
         }
 
+        private static void DecorateExceptionWithRuntimeErrorProperties(MsalException exception, Error runtimeError)
+        {            
+            var result = new Dictionary<string, string>()
+            {
+                { "RuntimeContext", runtimeError.Context },
```

Comment:
In public facing API, I don't think we should use the word `Runtime` - it's a bit generic and means something specific to us. Makes sense to use it in internal code, like Runtime WAM vs Legacy WAM; but in public facing code this should be something like `BrokerErrorContext`, `BrokerErrorTag`.

Reply:
But we could use the runtime for non-broker flows.